src/main/scala/chisel3/tester/package.scala

```diff
@@ -39,11 +40,19 @@ package object tester {
         require(x.binaryPoint == value.binaryPoint, "binary point mismatch")
         pokeBits(x, value.litValue)
       }
-      case (x: Bundle, value: Bundle) => {
+      case (xx: Bundle, value: Bundle) => {
         // TODO: chisel needs to expose typeEquivalent
-        require(x.elements.keys == value.elements.keys)  // TODO: this discards type data =(
-        (x.elements zip value.elements) foreach { case ((_, x), (_, value)) =>
-          x.poke(value)
+        require(xx.elements.keys == value.elements.keys)  // TODO: this discards type data =(
+        (xx.elements zip value.elements) foreach { case ((fieldName, xxx), (_, value)) =>
+          try {
+            xxx.poke(value)
+          }
+          catch {
+            case _: NoSuchElementException =>
+              val bundleFieldName = x.toString.split("[(]").head + "." + fieldName
```

Comment:
What is this weird code doing? It seems it's dependent on a debugging representation which isn't guaranteed to be stable. I'd prefer stability over making the display super pretty, unless it's unusable or something without this split thingy.

Reply:
Hack to get get bundle name.  I'm sure there's a better way